pkg/operator/ceph/csi/spec.go

```diff
@@ -155,6 +155,14 @@ const (
 	// update strategy
 	rollingUpdate = "RollingUpdate"
 	onDelete      = "OnDelete"
+
+	// driver daemonset names
+	RBDPlugin    = "csi-rbdplugin"
+	CephFSPlugin = "csi-cephfsplugin"
+
+	// driver deployment names
+	RBDProvisioner    = "csi-rbdplugin-provisioner"
+	CephFSProvisioner = "csi-cephfsplugin-provisioner"
```

Comment:
why these variable are exposed? is it getting used outside of this package?

Reply:
> why these variable are exposed? is it getting used outside of this package?

So that the creation and deletion will use the same consts [according to this discussion ](https://github.com/rook/rook/pull/5605#discussion_r446280313).

Reply:
Yeah it can be a const but we don't need to expose it, this is getting used in only spec package.